Add new module - Internal Developer Portal (Beta) (#1567)
harness/developer-hub: docs/internal-developer-portal/intro.md

---
title: Overview
description: A home for developers to create, manage, and explore software.
sidebar_position: 50
helpdocs_topic_id:
helpdocs_category_id:
helpdocs_is_private: false
helpdocs_is_published: true
---

# Overview of Harness IDP

:::note
This module is currently available in Beta to limited customers. Contact Harness Support if you would like to try it out.
:::

> Harness IDP lets you **create** new software components in seconds adhering to your company’s best practices. It lets you **manage** the software you own by presenting a developer-centric view of all the relevant information like service health, deployments and alerts. And it lets you **explore** the internal software ecosystem of the company, discover technical docs, APIs, services to enable better collaboration.

Reviewer: ```suggestion
Harness IDP enables you to create new software components quickly while adhering to your company’s best practices. It enables you to manage the software you own by presenting a developer-centric view of all relevant information such as service health, deployments, and alerts. It also enables you to explore the internal software ecosystem of the company, discover technical documentation, APIs, and services, all of which enable better collaboration.
```
* Can we back up the "in seconds" claim, or would the time taken depend on variables? I would prefer "Harness IDP enables rapid development of ...", but in the current sentence, "quickly" works too (still subjective but better than "in seconds").
* When someone says "lets you", our thoughts go to permission. "Enables you" is better. It means "gives you the ability to" or "empowers you" in a way. I hope these suggestions can be implemented. 
* Let's run with this description. In the next iteration, let's bring in the personas involved. The first part of this description is for the Platform Engineering team and the last part is for development teams, I think. Both user segments should be able to connect with this text. Thanks.

> Fully integrated into the Harness platform, IDP reduces the maintenance overhead and investment required from the platform engineering and is enterprise ready.

Reviewer: ```suggestion
Fully integrated into the Harness platform, IDP reduces the maintenance overhead and investment required from platform engineering teams and is enterprise ready.
```

---
title: Harness IDP Overview
description: A home for developers to create, manage and explore software
sidebar_position: 50
helpdocs_topic_id:
helpdocs_category_id:
helpdocs_is_private: false
helpdocs_is_published: true
---

> 

Reviewer: # Overview of Harness IDP

:::note
This module is currently available in Beta to limited customers. Contact Harness Support if you would like to try it out.
:::

> Harness IDP lets developers **create** new software components in seconds following the company’s best practices. It lets them **manage** their software by presenting a developer-centric view of all the relevant information. And it lets them **explore** the internal software ecosystem of the company, discover tech docs, APIs, services to enable better collaboration.

Reviewer: I would like to ask a question or two here. I definitely have suggestions for this text.

> Fully integrated into the Harness platform, IDP reduces the maintenance overhead and investment required from customers and is enterprise ready with our platform features.

Reviewer: Same as previous: I would like to ask a couple of questions here.
